### ISSUE TITLE
Can I change opennmt-py model to opennmt-tf model?

### ISSUE THREAD

**Issue by 520jefferson** [OpenNMT/OpenNMT-tf#937]
I want to do some knowledge distillation. The origin codes is pytorch,  I can use it to migrate to opennmt-py code related model, and then transfer to opennmt-tf model. Can I change opennmt-py model to opennmt-tf model?

**Comment by guillaumekln** [comment]
Why not run the knowledge distillation with OpenNMT-py?

There are no scripts to convert an OpenNMT-py model to OpenNMT-tf. You should manually map the model weights, as done in this issue for example: https://github.com/OpenNMT/OpenNMT-tf/issues/905#issuecomment-982218912.

**Comment by 520jefferson** [comment]
Yes, i will run the knowledge distillation from T5 big model(pytorch codes)  to OpenNMT-py model. Becasue the online service model is tf serving (pb file) related system, then i want to change  the openNMT-py model to openNMT-tf model, then i can do some  optimization on tf model before service launch.

**Comment by guillaumekln** [comment]
If your serving platform requires a TensorFlow model, you might as well run the distillation training with OpenNMT-tf directly. It should be much easier than trying to convert the model.

I'm closing this issue since the original question has been answered (the conversion is technically possible but is a bit tricky).

**Comment by 520jefferson** [comment]
If the small tf model  train and distill,   is it easy to transfer the data from tf model to pytorch  model when getting the logits of pytorch model ?    
I try with the tf.estimator and modify the model_fn and print the features and labels, i get nothing.
